.tools/runtimes/python.Makefile

```diff
@@ -1,3 +1,5 @@
+.PHONY: env install lint format check
```

Comment:
In this file is `env` really phony?

Or should env be kept phony and empty and then made dependent on the real venv files? (to keep the "abstraction" in-between)

Eg:

```
env: venv/bin/activate

venv/bin/activate:
# steps originally under env
```

Reply:
That is a good point. I can see doing that actually. I will add that as well